### DISCUSSION TITLE
Radeon ProRender 2.2.20 for Houdini and USD

### DISCUSSION THREAD

**Discussion by bsavery** [GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD#524] [Announcements]
- Support of Houdini 19.  Houdini 19 adds MaterialX nodes which Radeon ProRender supports through the plug-in.
- Updates to the Render Quality modes: 
    - RPR Interactive is a new mode supporting GPUs which use the Vulkan ray tracing extension, and is optimized for fast viewport rendering;
    - RPR Final, previously Full” mode. It is intended “” for final rendering the utmost physical correctness and image quality;

<hr /><em>This discussion was created from the release <a href='https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD/releases/tag/v2.2.20'>Radeon ProRender 2.2.20 for Houdini and USD</a>.</em>

**Comment by sbgraphic** [comment]
Hello,

Yesterday I tried Radeon ProRender in Houdini 19.0.x on my iMacPro, Vega 64X, 16 Go (Mac osx Big Sur). Installation process on OSX was quite painful for a newbie like me in Houdini (not much information on the web for mac users)… but after a few hours… I finally get this to "work".

However, I have a few questions, comparing to Blender 3.x (or even my "old" C4D R21 version), the preview on Houdini is soooo slow and sooo laggy… Is there a way to optimise this in Houdini on Mac with my configuration ?  I've also noticed that RPR 2.2.20 seems not compatible with 19.5.x, is it correct ? (i get crashes…).

Thanks in advance, if you can point me to the right direction ;-)
Best, Eric
